devicon.json

```
@@ -2569,7 +2599,28 @@
         "color": "#F7931E",
         "aliases": []
     },
-    {
+	{
+        "name": "msdos",
+        "tags": [
+            "os",
+        ],
+        "versions": {
+            "svg": [
+                "original",
```

Please remove the `,`
```suggestion
                "original"
```